app/controllers/concerns/scholars_archive/works_controller_behavior.rb

```diff
@@ -12,8 +12,32 @@ def edit
       super
     end
 
+    def update
+      set_other_option_values
```

Comment:
I feel like both #update and #create would be better suited as `before_action` callbacks and to set_other_option_values?